js/components/portal-dashboard/num-students-container.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import css from "../../../css/portal-dashboard/num-students.less";
+
+  interface IProps {
+    studentCount: number;
+  }
+
+export class NumberOfStudentsContainer extends React.PureComponent<IProps> {
+    render() {
+      return (
+        <div className={css.numStudentsContainer} data-test="num-student-container">
+          <span className={css.containerLabel}>Class: </span>
+          <span data-test="student-number">{this.props.studentCount}</span>
```

Comment:
Same comment as above, should we use `data-cy` to be consistent with the rest of the project?